tsconfig.strict.json

```diff
@@ -337,6 +337,8 @@
     "lib/manager/terragrunt/extract.ts",
     "lib/manager/terragrunt/index.ts",
     "lib/manager/terragrunt/modules.ts",
+    "lib/manager/tool-versions/extract.ts",
+    "lib/manager/tool-versions/index.ts",
```

Comment:
Can this be avoided?

Reply:
I can try if you request. But I'm new strict null checks, so can't say if that is easy or difficult to achieve.

Reply:
Maybe @zharinov can advice here?

Reply:
It depends on files that are far from null safety at the moment, so it would be easier to exclude for now. Though, null-safe code inside `tool-versions` folder is welcome.